src/xunit.runner.utility/Project/XunitFilters.cs

```diff
@@ -53,9 +60,22 @@ public bool Filter(ITestCase testCase)
                 return false;
             if (!FilterExcludedTraits(testCase))
                 return false;
+            if (!FilterIncludedNameSpaces(testCase))
+                return false;
 
             return true;
         }
+        bool FilterIncludedNameSpaces(ITestCase testCase)
+        {
+            // No assemblies in the filter == everything is okay
+            if (IncludedNameSpaces.Count == 0)
+                return true;
+
+            if (IncludedNameSpaces.Count != 0 && IncludedNameSpaces.Any(a => testCase.TestMethod.TestClass.Class.Name.Contains(a)))
```

Comment:
I'm not quite sure how I feel about this as an implementation for "namespace" filter. It's essentially a "partial name" filter.